pom.xml

```diff
@@ -637,6 +637,28 @@
                 </resources>
             </build>
         </profile>
+        <profile>
+            <id>mtr</id>
+            <activation>
+                <property>
+                    <name>downstream</name>
+                    <value>mtr</value>
+                </property>
+            </activation>
+            <properties>
+                <distribution.brand.name>Migration Toolkit for Runtimes by Red Hat (MTR)</distribution.brand.name>
+                <distribution.brand.name-acronym>MTR</distribution.brand.name-acronym>
+                <distribution.brand.documentation.url>https://developers.redhat.com/products/mta/overview/</distribution.brand.documentation.url>
```

Comment:
We should also add these references to MTA to the ticket mentioned in the other comment I've made.

PD: as discussed, we can probably open another ticket to use this in the TFL template, or we can use the same one mentioned above

Reply:
Ticket created https://issues.redhat.com/browse/WINDUP-3462